package.json

```diff
@@ -79,19 +80,20 @@
     "react-router-config": "^4.4.0-beta.6",
     "react-router-dom": "^4.4.0-beta.6",
     "react-select": "^1.3",
-    "reactstrap": "^6.4.0",
+    "reactstrap": "^8.5.1",
     "redux": "^4.0.0",
     "redux-form": "^7.4.2",
     "redux-thunk": "^2.1.0",
     "resize-observer-polyfill": "^1.5.0",
     "toposort": "^1.0.3",
     "url": "^0.11.0",
+    "uuid": "^8.2.0",
     "validator": "^6.1.0",
     "webpack-sources": "^1.1.0"
   },
   "devDependencies": {
     "@silverstripe/eslint-config": "^0.0.5",
-    "@silverstripe/webpack-config": "^1.5.0",
+    "@silverstripe/webpack-config": "^1.6.0",
```

Comment:
This will need to be incremented to 1.7 once the webpack PR is merged and release.